charts/karpenter/values.yaml

```diff
@@ -0,0 +1,77 @@
+nameOverride: ""  # Override the default name, empty means no override
+fullnameOverride: ""  # Override the full name, empty means no override
+
+# -- Global log level, defaults to 'info'
+logLevel: info
+# -- Log outputPaths - defaults to stdout only
+logOutputPaths:
+  - stdout
+# -- Log errorOutputPaths - defaults to stderr only
+logErrorOutputPaths:
+  - stderr
+
+# -- Additional annotations to add into metadata.
+additionalAnnotations: {}  # Empty dictionary for extra annotations
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+
+alicloud:
+  profiles:
```

Comment:
im not sure if there is a better way, the struct just like https://github.com/Vacant2333/karpenter-provider-alicloud/blob/69c2d8802ede58479faf50f18ce368a8dc0126d3/vendor/github.com/aliyun/aliyun-cli/config/configuration.go#L34

Reply:
Let's go with this temporarily. We will not use aksk in the future